filesystem/Folder.php

```diff
@@ -501,9 +230,71 @@ public function CMSTreeClasses() {
 	 * @return string
 	 */
 	public function getTreeTitle() {
-		return $treeTitle = sprintf(
+		return sprintf(
 			"<span class=\"jstree-foldericon\"></span><span class=\"item\">%s</span>",
-			Convert::raw2xml(str_replace(array("\n","\r"),"",$this->Title))
+			Convert::raw2att(Convert::nl2os($this->Title, ' '))
```

Comment:
nl2os doesn't _remove_ "\n" and "\r", it just converts them to the operating system default. Is that the change you intended?